pkg/kubefed2/federate/federate.go

```diff
@@ -44,9 +44,13 @@ import (
 const (
 	createResourceRetryTimeout  = 10 * time.Second
 	createResourceRetryInterval = 1 * time.Second
+	federationKindPrefix        = "Federated"
 )
 
 var (
+	// Controller created resources should always be skipped while federating content
+	controllerCreatedAPIResourceNames = []string{"events", "endpoints", "propagatedversions"}
```

Comment:
I don't think it's sufficient to use an unqualified name.  There is nothing stopping a 3rd party type from using the same name with a different group, so group-qualified is suggested.

Reply:
Updated.